setup.py

```diff
@@ -39,6 +39,26 @@
     ).stdout.split()[0][2:]] # get the first argument which is the library path
 
 
+if platform.system() == 'Windows':
+    compiler_args = ['/std:c++20']
+    compiler_and_linker_args = []
+    if 'NATCAP_INVEST_GDAL_LIB_PATH' not in os.environ:
+        raise RuntimeError(
+            'env variable NATCAP_INVEST_GDAL_LIB_PATH is not defined. '
+            'This env variable is required when building on Windows. If '
+            'using conda to manage your gdal installation, you may set '
+            'NATCAP_INVEST_GDAL_LIB_PATH="$CONDA_PREFIX/Library".')
+    library_dirs = [f'{os.environ["NATCAP_INVEST_GDAL_LIB_PATH"]}/lib']
+    include_dirs.append(f'{os.environ["NATCAP_INVEST_GDAL_LIB_PATH"]}/include')
```

Comment:
I think this is missing some recent changes from @davemfish . 

https://github.com/natcap/invest/blob/808a69796409ef72d32972e7e350fb719ffee89e/setup.py#L28

Reply:
@emilyanndavis removed this duplicated block in #1945, which is now merged!